docs/intro.rst

```diff
@@ -63,35 +69,58 @@ The ``mbdump-derived.tar.bz2`` archive contains annotations, user tags and searc
 These archives include all the data required for setting up an instance of
 CritiqueBrainz.
 
-You can automatically download the archives (``mbdump.tar.bz2`` and ``mbdump-derived.tar.bz2``) and
-begin the import for the MusicBrainz database::
+There are two ways in which you can import the database dump:-
 
-   $ docker-compose -f docker/docker-compose.dev.yml run musicbrainz_db
+1. By downloading and importing the database dump simultaneously::
```

Comment:
I'm not sure if simultaneously is the correct word here. Maybe something like `by downloading and importing the data in a single command`.